src/portable/instance/upgrade.rs

```diff
@@ -6,7 +6,7 @@ use std::time::SystemTime;
 use anyhow::Context;
 use const_format::concatcp;
 use fn_error_context::context;
-use gel_cli_derive::IntoArgs;
+use gel_cli_instance::cloud::CloudInstanceUpgrade;
```

Comment:
Seems to be a merge issue:

```suggestion
use gel_cli_derive::IntoArgs;
use gel_cli_instance::cloud::CloudInstanceUpgrade;
```

Reply:
I failed to merge using the web UI. Will push a fixed branch :)